app/models/users/student.rb

```diff
@@ -14,4 +18,17 @@ def self.is_student?(data)
     Student.where(email: data["email"]).first
   end
 
+  def self.lost
+    result = []
+    result << joins(events: [:scans] ).where(at_school: true)
+                                      .where("start_time <= ? AND end_time >= ?", Time.now, Time.now )
+                                      .where.not(scans:{correct: true})
+    joins(:events).where(at_school: true)
+                  .where("start_time <= ? AND end_time >= ?", Time.now, Time.now).each do |student|
+                    result << student if student.events.last.scans.count == 0
+                  end
+
+    result.flatten
```

Comment:
i don't see these tests under the `model/student_spec` should I be looking elsewhere? Or do you mean, you are currently testing them and haven't pushed?